--- mod/common/buildings/Temple-Manager-Repellent.txt	2023-09-12 12:30:00.621240100 -0400
+++ mod/common/buildings/Temple-Manager-Repellent.txt.work	2023-09-12 12:29:59.389756100 -0400
@@ -79,21 +79,10 @@
 				is_worker_coop_empire = no
 			}
 		}
 		modifier = {
-			job_manager_add = 1
-		}
-	}
-
-	triggered_desc = {
-		trigger = {
-			exists = owner
-			owner = {
-				is_megacorp = yes
-				is_worker_coop_empire = no
-			}
+			job_priest_add = 1
 		}
-		text = job_manager_effect_desc
 	}
 
 	triggered_planet_modifier = {
 		potential = {
@@ -222,21 +211,10 @@
 				is_worker_coop_empire = no
 			}
 		}
 		modifier = {
-			job_manager_add = 2
-		}
-	}
-
-	triggered_desc = {
-		trigger = {
-			exists = owner
-			owner = {
-				is_megacorp = yes
-				is_worker_coop_empire = no
-			}
+			job_priest_add = 2
 		}
-		text = job_manager_effect_desc
 	}
 
 	triggered_planet_modifier = {
 		potential = {
@@ -373,21 +351,10 @@
 				is_worker_coop_empire = no
 			}
 		}
 		modifier = {
-			job_manager_add = 3
-		}
-	}
-
-	triggered_desc = {
-		trigger = {
-			exists = owner
-			owner = {
-				is_megacorp = yes
-				is_worker_coop_empire = no
-			}
+			job_priest_add = 3
 		}
-		text = job_manager_effect_desc
 	}
 
 	triggered_planet_modifier = {
 		potential = {
@@ -578,21 +545,10 @@
 				is_worker_coop_empire = no
 			}
 		}
 		modifier = {
-			job_manager_add = 2
-		}
-	}
-
-	triggered_desc = {
-		trigger = {
-			exists = owner
-			owner = {
-				is_megacorp = yes
-				is_worker_coop_empire = no
-			}
+			job_priest_add = 2
 		}
-		text = job_manager_effect_desc
 	}
 
 	triggered_planet_modifier = {
 		potential = {
@@ -706,21 +662,10 @@
 				is_worker_coop_empire = no
 			}
 		}
 		modifier = {
-			job_manager_add = 2
-		}
-	}
-
-	triggered_desc = {
-		trigger = {
-			exists = owner
-			owner = {
-				is_megacorp = yes
-				is_worker_coop_empire = no
-			}
+			job_priest_add = 2
 		}
-		text = job_manager_effect_desc
 	}
 
 	triggered_planet_modifier = {
 		potential = {
@@ -890,10 +835,9 @@
 				is_spiritualist = yes
 			}
 		}
 		modifier = {
-			job_manager_add = 1
-			job_priest_add = 1
+			job_priest_add = 2
 		}
 	}
 
 	triggered_planet_modifier = {
@@ -1004,5 +948,5 @@
 
 	prerequisites = {
 		"tech_secrets_league"
 	}
-}
\ No newline at end of file
+}
